frontend/Makefile

```diff
@@ -18,7 +18,7 @@ lint:
 
 .PHONY: mypy
 mypy:
-	mypy .
+	mypy --ignore-missing-imports --follow-imports=skip --strict-optional --warn-no-return .
```

Comment:
any reason for adding this?

Reply:
I actually didn't add this (as seen here https://github.com/amundsen-io/amundsen/blob/main/frontend/Makefile#L21), as part of this PR I had moved the logic under the mypy section of the setup.cfg file but for some reason the `make mypy` command wasn't picking up all of the configs so I reverted it back to what it was before.